### PR TITLE
Migration of react-leaflet to version 3 & MapScale

### PR DESCRIPTION
**Pull request by vlach1989**
See MapScale functionality at: https://gisat.github.io/app-docs/ Components > Maps > Map scale

### CODE REVIEW

File: src/ReactLeafletMap/layers/WmsLayer/SingleTileLayer.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import {createLayerComponent} from '@react-leaflet/core';
+import wms from './leaflet.wms';
+
+function createLeafletElement(props, context) {
+	const {pane, url, crs, params} = props;
```
Comment (vlach1989) [inline src/ReactLeafletMap/layers/WmsLayer/SingleTileLayer.js:6]: Where does pane prop come from?

File: src/ReactLeafletMap/layers/WmsLayer/index.js [hunk 2]
```diff
@@ -42,22 +42,33 @@ const getFinalParams = (params, opacity) => {
...
 	);
 
-	return (
-		<WMSTileLayer
-			key={layerKey || i}
-			url={options.url}
-			crs={crs ? projectionHelpers.getCRS(crs) : null}
-			// singleTile={o.singleTile === true} TODO single tile layer
-			params={finalParams}
+	if (options.singleTile) {
```
Comment (vlach1989) [inline src/ReactLeafletMap/layers/WmsLayer/index.js:52]: `const {params, singleTile} = options`
....
`if (singleTile)`

File: src/ReactLeafletMap/layers/WmsLayer/SingleTileLayer.js (hunk 1 above)
Comment (vdubr) [reply src/ReactLeafletMap/layers/WmsLayer/SingleTileLayer.js:6]: It was part of magic who I found. Pane is injected by ReactLeaflet.

**Comment by vlach1989** [comment]
Outdated

**Pull request closed without merging**